src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -337,6 +358,198 @@ public string GetFileShareLocationForReleaseJson(DownloadedBuild build)
             return Path.Combine(build.OutputDirectory, shippingSubPath);
         }
 
+        const string githubRepoPrefix = "https://github.com/";
+        const string azdoRepoPrefix = "https://dev.azure.com/dnceng/internal/_git/";
+        const string coreRepoCategory = "core";
+        const string aspnetCategory = "aspnet";
+        const string wcfCategory = "wcf";
+        
+        // The following is the list of repos that should be picked up by the tooling
+        // This list is effectively static, but not the full set of repos that are in the graph,
+        // as our release process does not publish all of the packages (e.g. not nuget.client).
+        // Each dictionary entry maps to a tuple of the category of the package (owner) and short name of the repo.
+        private static readonly Dictionary<string, (string, string)> repositories = new Dictionary<string, (string, string)>()
+        {
+            // Core
+
+            // Public
+            { $"{githubRepoPrefix}dotnet/corefx", (coreRepoCategory, "corefx") },
+            { $"{githubRepoPrefix}dotnet/coreclr", (coreRepoCategory, "coreclr") },
+            { $"{githubRepoPrefix}dotnet/core-setup", (coreRepoCategory, "core-setup") },
+            // Internal
+            { $"{azdoRepoPrefix}dotnet-corefx", (coreRepoCategory, "corefx") },
+            { $"{azdoRepoPrefix}dotnet-coreclr", (coreRepoCategory, "coreclr") },
+            { $"{azdoRepoPrefix}dotnet-core-setup", (coreRepoCategory, "core-setup") },
```

Comment:
Is core-setup correct? I thought all of this collapsed into /runtime?

Reply:
Ah, we need dotnet/runtime and dotnet-runtime in addition to these. Nothing was collapsed before 5.0.

Reply:
ahh yes. I'll add.